Dockerfile.x86

```diff
@@ -0,0 +1,113 @@
+FROM multiarch/alpine:x86-v3.7
+MAINTAINER wessel<jimfromsa@gmail.com>
+
+ENV IOP_VERSION  v1.2-Beta
+ENV IOP_PLATFORM linux-64
+
+
+# Ports for node2node, client2node, interprocess communication between local services
+EXPOSE 16980
+EXPOSE 16981
+EXPOSE 16982
+
+
+# Public node id, should be an SHA256 hash of the public key of this node
+#ENV NODEID
+
+# TCP port to serve client (i.e. end user) queries. Optional,default value: 16981
+ENV CLIENTPORT 16981
+
+# TCP port to serve other IoP services running on this node. Optional, default value: 16982
+ENV LOCALPORT 16982
+
+#  TCP port to serve node to node communication. Optional, default value: 16980
+#Should only be locally available
```

Comment:
The comment on local availability is wrong here, it should stand with the LocalPort as its name also clearly states.